container/lxd/lxd.go

```diff
@@ -132,21 +130,21 @@ func (manager *containerManager) CreateContainer(
 	}
 
 	callback(status.Running, "Container started", nil)
-	return &lxdInstance{name, manager.server},
```

Comment:
All of these renames follow the same 'issue'. Which is that Juju code tends to call the local side of the connection 'client' but LXD wants you to call your connection to the server Server.
I thought the goal was to be more consistent in the LXD handling code to follow the lxc/lxd conventions.